01_fib/kasunfdo/main.go

```diff
@@ -1,36 +0,0 @@
-package main
```

Comment:
I believe (I hope I'm not wrong), the idea is that once each persons lab implementation is approved, it gets merged into master branch, and so it will show up in your future branches.

If we merge your branch now, this persons work will be deleted.

I suggest you rebase and drop this commit, that drops this other contributors code.

Reply:
I think i fixed this by rebasing my branch :)